packages/flutter/lib/src/painting/decoration.dart

```diff
@@ -198,9 +198,18 @@ abstract class Decoration with Diagnosticable {
 /// happens, the [onChanged] callback will be invoked. To stop this callback
 /// from being called after the painter has been discarded, call [dispose].
 abstract class BoxPainter {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const BoxPainter([this.onChanged]);
```

Comment:
This is a user-visible breaking change.

I don't think we should do this.

Reply:
Ok, then I will instrument all descendants.